words.html

```diff
@@ -514,10 +514,10 @@
 ...die <span class="tft">Nachäffer</span> erben die Welt.
 </p>
 <p id="evo_9_all_c">
-Also, obwohl du daneben gelegen hast -- die freundlichen-aber-naiven <span class="all_c">Immer Kooperieren</span> waren von Anfang an verdammt -- hat sich am Ende eine <i>kluge</i> Art von Freundlichkeit durchgesetzt, und die <span class="all_d">Immer Betrügen</span> wurden zerquetscht.
+Also, obwohl du daneben gelegen hast – die freundlichen-aber-naiven <span class="all_c">Immer Kooperieren</span> waren von Anfang an verdammt -- hat sich am Ende eine <i>kluge</i> Art von Freundlichkeit durchgesetzt, und die <span class="all_d">Immer Betrügen</span> wurden zerquetscht.
```

Comment:
Die doppelten Bindestriche sind eigentlich ein [Gedankenstrich](http://www.duden.de/sprachwissen/rechtschreibregeln/gedankenstrich) (-- = –) und fast immer als Einschub mit besonderer Abgrenzung zum restlichen Text gedacht. Deutsch und Englisch unterscheiden sich da nicht wirklich und ich will möglichst nah am Original bleiben in dem auch "--" statt "–" verwendet wird. Daher bitte alle Änderungen an den Strichen zurücknehmen.